core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1245,6 +1245,28 @@ trait ZIOFunctions extends Serializable {
    */
   final def unsandbox[R >: LowerR, E <: UpperE, A](v: ZIO[R, Cause[E], A]): ZIO[R, E, A] = v.catchAll[R, E, A](halt)
 
+  /**
+   * Returns the identity effectful function, which performs no effects
+   */
+  final def identity[A >: LowerR]: ZIO[A, Nothing, A] = fromFunction[A, A](a => a)
```

Comment:
Is this the proper return type? 
scala```
ZIO[A, Nothing, A]
```

Reply:
It was in the `FunctionIO` and `fromFunction` signature is `fromFunction[R >: LowerR, A](f: R => A): ZIO[R, Nothing, A] = ...`

Reply:
Yes, this is correct. Function from `A => A`.